docs/static/troubleshoot/ts-azure.asciidoc

```diff
@@ -0,0 +1,81 @@
+[[ts-azure]]
+==== Azure Event Hub issues and solutions
+ 
+[discrete] 
+[[ts-azure-http]]
+===== Event Hub plugin can't connect to Storage blob (input)
+
+*Symptoms* 
+
+Azure EventHub plugin can't connect to the blob storage:
+
+-----
+[2024-01-01T13:13:13,123][ERROR][com.microsoft.azure.eventprocessorhost.AzureStorageCheckpointLeaseManager][azure_eventhub_pipeline][eh_input_plugin] host logstash-a0a00a00-0aa0-0000-aaaa-0a00a0a0aaaa: Failure while creating lease store
+com.microsoft.azure.storage.StorageException: The client could not finish the operation within specified maximum execution timeout.
+-----
+
+The plugin can't complete the registration phase because can't connect to the Azure Blob Storage configured
+in the plugin `storage_connection` setting.
+
+
+*Background*
+
+Azure Event Hub plugin can share the offset position of a consumer group with
+other consumers only if configures the Blob Storage connection setting.
+Despite EventHub uses the AMQP protocol to transfer data, the Blob Storage uses a
+library which leverages the JDK's http client, `HttpURLConnection`.
+To troubleshoot HTTP connection problems, maybe related to proxy settings, the
```

Comment:
To troubleshoot HTTP connection problems, which may be related to proxy settings, the